conversation.py

```diff
@@ -6,13 +8,16 @@ def __init__(self, game, engine, xhr):
 
     command_prefix = "!"
 
-    def react(self, line):
+    def react(self, line, game):
         print("*** {} [{}] {}: {}".format(self.game.url(), line.room, line.username, line.text.encode("utf-8")))
         if (line.text[0] == self.command_prefix):
-            self.command(line, line.text[1:].lower())
+            self.command(line, game, line.text[1:].lower())
         pass
 
-    def command(self, line, cmd):
+    def command(self, line, game, cmd):
+        if cmd == "wait":
+            game.abort_at = time() + 60
+            self.send_reply(line, "Waiting 60 seconds...")
```

Comment:
the feedback is a good idea, but it should only be sent when the game is actually abortable.